src/Model/Linsen.hs

```diff
@@ -25,8 +25,8 @@ import           Data.List.Extra                          ( (!?) )
 import           Data.Text.Lazy                           ( Text
                                                           , replace
                                                           , strip )
-import           Data.Thyme                               ( parseTime
-                                                          , defaultTimeLocale )
+import           Data.Thyme                               ( parseTime )
+import           Data.Time.Format                         (TimeLocale (..) )
```

Comment:
Can you use https://hackage.haskell.org/package/thyme-0.4/docs/Data-Thyme-Format.html#t:TimeLocale instead? So you don't have to add another dependency.

Reply:
So, about that

Reply:
You see, Thyme uses Time as the dependency, which this constructor, but yes, it does reexport it so I will change that

Reply:
Sweet!